MSAL/test/app/ios/MSALTestAppAcquireTokenViewController.m

```diff
@@ -433,10 +501,25 @@ - (void)acquireTokenInteractive:(id)sender
                 [self updateResultViewError:error];
             }
             
+            [_webView loadHTMLString:@"<html><head></head><body>done!</body></html>" baseURL:nil];
+            [_authView setHidden:YES];
+            [self.view setNeedsDisplay];
+            
             [[NSNotificationCenter defaultCenter] postNotificationName:MSALTestAppCacheChangeNotification object:self];
         });
     };
     
+    application.webviewType = _webviewSelection.selectedSegmentIndex == 0 ? MSALWebviewTypeWKWebView : MSALWebviewTypeAutomatic;
```

Comment:
so, we only have 2 options in the test app?

Reply:
For now, yes. just embedded and systemAuto.